1.1/translations/sf_de.ttl

```diff
@@ -0,0 +1,167 @@
+  skos:prefLabel "maximum"@de 
+.
+
+sf:minimum
+  skos:definition "Der Punkt mit den maximalen Koordinatenwerten einer Bounding Box"@de ;
+  skos:prefLabel "minimum"@de 
+.
+
+sf:Geometry 
+    skos:prefLabel "Geometrie"@de ;
+    skos:definition """Die Klasse Geometrie die ist die Ursprungslasse der Klassenhierarchie.
+
+Die instanziierbaren Unterklassen der Klasse geometrie sind auf 0, 1 und 2-dimensionale geometrische Objekte in einem 2, 3 oder vierdimensionalem Koordinatenraum (R2,R3,R4) begrenzt. Geometrien in R2 haben einen X und einen Y Koordinatenwert. Geometiren in R3 haben X, Y und Z Koordinatenwerte oder X, Y und M Koordinatenwerte. Geometrien in R4 beinhalten Punkte mit Koordinatenwerten für X, Y, Z und M.
+
+Die Interpretation der Koordinaten erfolgt mittels des Koordinatenreferenzsystems welches der jeweiligen Geometrie zugeordnet wurde. Alle Koordinaten innerhalb eines geometrischen Objekts sollten im gleichen Koordinatenreferenzsystem definiert sein. Jede Koordinate soll eindeutig zu einem Koordinatenreferenzsystem bezogen sein. Dies kann durch einen direkten Verweis oder über die beschreibende Geometrie geschehen. Die Z Koordinate einer Geometrie repräsentiert typischerweise, aber nicht notwendigerweise eine Erhebungsgröße oder eine Höhenbeschreibung. Die M Koordinate repräsentiert einen Messwert.
+
+Alle Geometrieklassen in dieser Spezifikation sind so definiert, dass ihre Instanzen topologisch geschlossen sind. Dies bedeutet, dass alle Geometrien ihre Begrenzungen als Punktmenge mitbeinhalten. Dies hat keinen Einfluss auf ihre Repräsentierung und topologisch nicht geschlossene Geometrien könnten durch ähnliche Klassen modelliert werden falls die Notwendigkeit besteht."""@de .
+
+sf:GeometryCollection 
+    skos:prefLabel "Geometriesammlung"@de ;
+    skos:definition """Eine Sammlung von Geometrien ist ein geometrisches Objekt die eine Sammlung von weiteren nicht notwendigerweise gleichartigen anderen geometrischen Objekten beinhaltet.
+
+Alle Elemente einer Geometriesammlung sollen im gleichen räumlichen Bezugssystem definiert sein. Man bezeichnet dieses Bezugssystem das räumliche Bezugssystem der Geometriesammlung.
+Die Geometriesammlung definiert keine weiteren Beschränkungen in Bezug auf die in ihr enthaltenen geometrischen Objekte. Unterklassen der Klasse sf:GeometryCollection können weitere Restriktionen basierend auf der räumlichen Dimension oder auf der Basis von Beziehungen der Geometrien untereinander definieren."""@de .
+
+sf:Line
+    skos:prefLabel "Linie"@de ;
+    skos:definition """Eine Linie ist ein LineString mit genau zwei Punkten."""@de .
+
+sf:LineString 
+    skos:prefLabel "LineString"@de ;
+    skos:definition """Ein sf:LineString ist eine Kurve mit einer linearen Interpolation zwischen ihren zugehörigen Punkten. Jedes aufeinanderfolgende Paar von Punkten definiert ein Liniensegment."""@de .
+
+sf:LinearRing 
+    skos:prefLabel "Linearer Ring"@de ;
```

Comment:
Some of these prefLabels have spaces, like this one, and some don't like "LineString". Can you stnadardise on one form or the other? I prefer the spaces!